tests/Unit/SnowflakeTest.php

```diff
@@ -3,15 +3,14 @@
 namespace Tests\Unit;
 
 use Tests\TestCase;
-use App\Services\SnowflakeService;
 
 class SnowflakeTest extends TestCase
 {
     /** @test */
     public function snowflakeTest()
     {
-    	$expected = 266077397319815168;
-    	$actual = 266077397319815168;
-    	$this->assertEquals($expected, $actual);
+        $expected = 266077397319815168;
+        $actual = 266077397319815168;
+        $this->assertEquals($expected, $actual);
```

Comment:
As far as I can tell, this doesn't actually _do_ anything? I kept the test in case it did something I'm not aware of, but just let me know and I'll remove it.